nutkit/protocol/requests.py

```diff
@@ -228,6 +228,16 @@ def __init__(self, initial_bookmarks,
         self.bookmarksConsumerRegistered = bookmarks_consumer_registered
 
 
+class BookmarkManagerClose:
+    """Destroy the bookmark manager in the backend and free the resources.
+
+    Backend should respond with a BookmarkManager response.
+    """
```

Comment:
It may worth emphasise this method is not a part of the BookmarkManager implementation, it is only a helper method for testkit resource management.